src/tests/suite/daos_nvme_recovery.c

```diff
@@ -66,13 +66,51 @@ nvme_fault_reaction(void **state, bool mode)
 	int			 n_tgtidx = 0;
 	int			 per_node_tgt_cnt = 0;
 	int			 i, j, k, rc;
-	int		 faulty_disk_idx = 0;
+	int			 faulty_disk_idx = 0;
+	uuid_t			 offline_pool_uuid;
 
 	if (!is_nvme_enabled(arg)) {
 		print_message("NVMe isn't enabled.\n");
 		skip();
 	}
 
+
+	/**
+	 * If test need multiple pool with both mode offline and online
+ 	 * create the another pool which will be offline by default.
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs